### DISCUSSION TITLE
How to show litee sidebar?

### DISCUSSION THREAD

**Discussion by eclbg** [ldelossa/litee.nvim#101] [Q&A]
I can't figure out how to show the side bar. For now I'm only attempting to make litee-calltree work without success on a python repository.
Apologies for the dumb question

**Comment by ldelossa** [comment]
No worries. Is "LTPanel" what youre looking for? Its a bit confusing since the "panel" is actually implemented in "litee.nvim" because its a shared component that other plugs can "register" their UI into. 

See: https://github.com/ldelossa/litee.nvim/blob/de1a3d65c0917bcb933ad023768f6b6e74f6ca92/doc/litee.txt#L250

and 

https://github.com/ldelossa/litee.nvim/blob/de1a3d65c0917bcb933ad023768f6b6e74f6ca92/lua/litee/lib/commands.lua#L5

**Reply by eclbg** [reply]
I'm not sure. Using Plug I've installed litee and litee-calltree and would like to see the call hierarchy of a method in my codebase. I also have the following lines in my .nvimrc which.

`lua << EOF
-- configure the litee.nvim library
require('litee.lib').setup({
    tree = {
        icon_set = "codicons"
    },
    panel = {
        orientation = "left",
        panel_size  = 30
    }
})
-- configure litee-calltree.nvim
require('litee.calltree').setup({})
EOF`

Now I don't know how to see the call hierarchy in the sidebar as shown in the example screenshot. 

Really appreciate your fast response! This feature looks terrific and I'm excited to get it to work.

**Reply by eclbg** [reply]
Running `:LTPanel` displays:
Must open a litee component before toggling the panel
I've tried running it both on a method definition and on a method call.

After looking at the list of available commands I've been trying with LTOpenToCalltree but it doesn't do anything nor displays an error message. Most other commands display the error "Must perform an call hierarchy LSP request first"

**Reply by ldelossa** [reply]
This plugin hijacks the native LSP handlers. So you need to actually issue an LSP "incoming" or "outgoing" calls LSP command. Then, the calltree plugins will be called instead and open the panel with the new application state 

If this sounds confusing, you may want to ":h lsp" (i think) to understand how that all works. Ideally, you use the nvim-lsp-config plugin to set up some key bindings for those LSP calls.
